e2e-tests/popup/settings/change-password.spec.ts

```diff
@@ -0,0 +1,86 @@
+import { popup, popupExpect } from '../../fixtures';
+
+popup.describe('Popup UI: change password', () => {
+  popup(
+    'should succesfully change password',
+    async ({
+      popupPage,
+      unlockVault,
+      providePassword,
+      changePassword,
+      provideNewPassword,
+      lockVault,
+      unlockVaultNewPassword
+    }) => {
+      await unlockVault();
+      await popupPage.getByTestId('menu-open-icon').click();
+      await popupPage.getByText('Change Password').click();
+      await providePassword();
+
+      await popupExpect(popupPage.getByText('Change Password')).toBeVisible();
+
+      await changePassword();
+
+      await popupPage.getByTestId('menu-open-icon').click();
+      await popupPage.getByText('Back up your secret recovery phrase').click();
+      await provideNewPassword();
+
+      await popupExpect(
+        popupPage.getByRole('heading', {
+          name: 'Back up your secret recovery phrase'
+        })
+      ).toBeVisible();
+      await lockVault();
+      await unlockVaultNewPassword();
+
+      await popupExpect(
+        popupPage.getByTestId('network-switcher').getByRole('img')
+      ).toBeVisible();
```

Comment:
why did you check this? it does not apply to this test

Reply:
I added it only to make sure that user is logged in